### DISCUSSION TITLE
Google Colab notebook

### DISCUSSION THREAD

**Discussion by AnalyticsXXI** [mcmonkeyprojects/SwarmUI#5] [Q&A] [answered]
Could anyone share Colab notebook to run SwarmUI?

**Comment by mcmonkey4eva** [comment, marked as answer]
It's in the readme https://github.com/mcmonkeyprojects/SwarmUI?tab=readme-ov-file#try-it-on-google-colab-or-runpod

**Reply by AnalyticsXXI** [reply]
Oh, right, sorry for being stupid :)
